controller/instance_manager_controller.go

```diff
@@ -545,7 +546,7 @@ func (imc *InstanceManagerController) createGenericManagerPodSpec(im *longhorn.I
 					LivenessProbe: &v1.Probe{
 						Handler: v1.Handler{
 							Exec: &v1.ExecAction{
-								Command: []string{"/usr/local/bin/grpc_health_probe", "-addr=:8500"},
+								Command: []string{"ls", filepath.Join(types.GetEngineBinaryDirectoryInContainerForImage(image.Spec.Image), "longhorn")},
```

Comment:
Reason?

Reply:
Make sure the engine binary is available on the **each** corresponding node. `types.EngineBinaryExistOnHostForImage(engineImage.Spec.Image)` can gurantee the binary is available on the node `ic.controllerID`.

Reply:
To deal with which scenario? The instance manager's lifecycle is already controlled by the engine image controller.

Reply:
All right. This modification is unnecessary. Will remove it.